dbms/src/Interpreters/Aggregator.cpp

```diff
@@ -1403,8 +1508,12 @@ struct AggregatorMethodInitKeyColumnHelper<AggregationMethodOneKeyStringNoCache<
 
     void initAggKeys(size_t rows, std::vector<IColumn *> & key_columns)
     {
-        Method::initAggKeys(rows, key_columns[0]);
         index = 0;
+        RUNTIME_CHECK_MSG(
+            key_columns.size() == 1,
```

Comment:
This check will always be true because if no key column need to be convert, `filler_skip_convert_key` will be called?

Reply:
Yes. Add this check only for guard